kahuna/package.json

```diff
@@ -32,6 +32,7 @@
       "text": "github:systemjs/plugin-text@^0.0.2",
       "theseus": "npm:theseus@0.5.2",
       "theseus-angular": "npm:theseus-angular@^0.3.1",
+      "titip": "github:quantumui/titip@master",
```

Comment:
I've merged your registry PR. I don't think versions have anything to do with the registry, but we should probably reference a Git hash until they setup a version we can reference in the upstream repo.